lib/eventmachine.rb

```diff
@@ -881,6 +881,19 @@ def self.open_datagram_socket address, port, handler=nil, *args
     c
   end
 
+  # Same as open_datagram_socket but also multicast subscribes
+  def self.open_datagram_socket_and_mcastjoin address, port, handler=nil, *args
+
+    klass = klass_from_handler(Connection, handler, *args)
+    s = EM::open_udp_socket address, port.to_i
+    ip =  IPAddr.new( address ).hton + IPAddr.new("0.0.0.0").hton
```

Comment:
I don't remember enough about multicast if the interface address should be configurable here or just use the any address?